A4_3.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+int+push;
+
+public class A4_3 {
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+
+		Scanner scan = new Scanner(System.in);
+		System.out.print("Enter a number greater than 0 (less than 1 to quit): ");
+		
+		int num = scan.nextInt();
+		int sum = 0;
+		int square;
+		
+		if(num > 1 || num == 1) {
+			for (int n =1 ; n < num+1 ; n++ ) {
+				square = n * n;

```

Comment:
you don't even need this. you could sum += n*n

Reply:
How can I make it, so it keeps looping asking to enter a new number?  only stopping completely when entering zero.

Reply:
like this
 "This program reads an integer greater than 0 from the user and then uses a loop to calculate the sum of the squares of the integers from 1 to that integer. For example, if the read integer equals 4, the sum of the squares is 30 (1 + 4 + 9 + 16). The program should repeat this process until the user wants to quit. An input value less than or equal to 0 signals the end of the data."

Reply:
I'll have to use a different/better loop in order to create a looping sequence right?